examples/react/tutorial/step1-login/README.md

```diff
@@ -1,18 +1,46 @@
+# Getting Started with Create React App
 
-## Overview
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-This example demonstrates the basics of using the ORE ID
-- Initializing the oreid-js library with your app's appId
-- Handling the login flow
-- Retrieving the user's info from the API
-- Using the LoginButton React component
-<br><br>
+## Available Scripts
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app)
+In the project directory, you can run:
 
-    yarn start
+### `npm start`
```

Comment:
If we are using `yarn` please update the commands here .

Reply:
Done!